### PR TITLE
feat(octavia): adding pool creation page

### PR DESCRIPTION
**Pull request by JacquesLarique**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/octavia-batch-1`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-12255, MANAGER-12610
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~ (n/a)

## Description

Added the pool creation page and fixed the request ID display in error messages

### CODE REVIEW

File: packages/manager/core/generator/index.js
Comment (ghyenne) [inline packages/manager/core/generator/index.js:1]: The empty file can be remove.

File: packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/pools/components/pool-form/index.js
Comment (darsene) [inline packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/pools/components/pool-form/index.js:1]: no need to create lazy loading for this module ?

File: packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/pools/create/template.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<div class="text-center" data-ng-if="$ctrl.loading">
+    <oui-spinner></oui-spinner>
+</div>
+
+<div data-ng-if="!$ctrl.loading">
+    <octavia-load-balancer-pool-form
```
Comment (darsene) [inline packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/pools/create/template.html:6]: `data-ng-if` could be set directly on `octavia-load-balancer-pool-form`. `<div>` is not really ncessary

File: packages/manager/core/generator/index.js
Comment (darsene) [inline packages/manager/core/generator/index.js:1]: What's that file for ?

**Review by qpavy — APPROVED** [review]

File: packages/manager/core/generator/index.js
Comment (JacquesLarique) [reply packages/manager/core/generator/index.js:1]: I don't know my ide see things git doesn't and it seems like I didn't check enough the changes I commited --'

File: packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/pools/create/template.html (hunk 1 above)
Comment (JacquesLarique) [reply packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/pools/create/template.html:6]: Strange I have a class="text-center" on this div that does not appear here O_o

File: packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/pools/components/pool-form/index.js
Comment (JacquesLarique) [reply packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/pools/components/pool-form/index.js:1]: It is lazy loaded through create/module.js (lazy loaded by create/index.js)

**Review by darsene — APPROVED** [review]

**Review by ghyenne — APPROVED** [review]

**Pull request merged**